### ISSUE TITLE
Touch through does not work on android with overlapping scroll views

### ISSUE THREAD

**Issue by LoganElliott** [chrisfisher/react-native-directed-scrollview#5]
In your example you have "row label" over the grey boxes but in android you can't click the boxes behind the labels. In IOS this works.

This could be an issue with your library or with how pointerevents for React Native on Android

**Comment by chrisfisher** [comment]
@LoganElliott Yep I know the issue you mean. I'll try take a look when I get some time, or else feel free to raise a PR 😄

**Comment by luisfuertes** [comment]
Any update of this?

**Comment by chrisfisher** [comment]
@luisfuertes No, sorry. PR's welcome :)

**Comment by WalkerCS** [comment]
hello, I have same issue any update on this issue?

**Comment by WalkerCS** [comment]
Hi Guys this issue had been solve just add zIndex:-1 in StyleSheet rowLabelsContainer in Grid.js

**Comment by WalkerCS** [comment]
Hi Guys this issue had been solve just change width: 45 in StyleSheet rowLabelsContainer in Grid.js
